### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by iamcarbon**
This PR is focused on improving code quality and does not include any functional changes.

### CODE REVIEW

**Comment by iamcarbon** [comment]
Ready for review

**Comment by iamcarbon** [comment]
@ping H1Gdev

**Comment by H1Gdev** [comment]
Do these changes cover the entire project ?
How did you find files to change ?

**Comment by iamcarbon** [comment]
Hi H1Gdev. These fixes were found using simple searches for common patterns that could be optimized while familiarizing myself with the code.

File: Source/DataTypes/EnumConverters.cs [hunk 1]
```diff
@@ -54,19 +54,19 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
         /// <summary>Attempts to convert the value to the destination type.</summary>
         public override object ConvertTo(ITypeDescriptorContext context, CultureInfo culture, object value, Type destinationType)
         {
-            if (destinationType == typeof(string) && value is T)
+            if (destinationType == typeof(string) && value is T s)
```
Comment (FoggyFinder) [inline Source/DataTypes/EnumConverters.cs:57]: why? `s` looks unused

File: Source/DataTypes/EnumConverters.cs [hunk 2]
```diff
@@ -54,19 +54,19 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
...
             {
                 var stringValue = ((T)value).ToString();
```
Comment (FoggyFinder) [inline Source/DataTypes/EnumConverters.cs:59]: replace with `s`?

Comment (iamcarbon) [reply Source/DataTypes/EnumConverters.cs]: done

**Comment by iamcarbon** [comment]
Going to go ahead and close this for now since these refactoring cause conflicts as changes get merged in -- and the history gets messy.

@H1Gdev Let me know if you're interested in any code cleanups to improve the maintainability of the library in the future.

**Pull request closed without merging**